spec/features/guest_user_spec.rb

```diff
@@ -68,10 +68,9 @@
         check 'company[hiring]'
         fill_in 'company[hire_count]', with: 5
         click_button('Create techHire account')
-        expect(current_path).to eq(root_path)
+        expect(current_path).to eq(company_path(Company.last.id))
```

Comment:
I'll update that in my new branch since this one only relates to creating a company and user upon signup.